### PR TITLE
v10.0.2

### PR DESCRIPTION
**Pull request by brendankenny**
patch release to get a handful of bug and correctness fixes published

- [x] will wait to include #14839
- [x] #14838 would be nice if a reviewer is so inclined :)

edit:
- [x] probably want #14800 as well if @paulirish is good with it

adam suggests
- [x] https://github.com/GoogleChrome/lighthouse/pull/14790

### CODE REVIEW

**Comment by adamraine** [comment]
Gotta bump the version in e2e tests now. Version update script doesn't cover it.

**Comment by adamraine** [comment]
Can we add https://github.com/GoogleChrome/lighthouse/pull/14790?

File: third-party/devtools-tests/e2e/lighthouse/navigation_test.ts [hunk 1]
```diff
@@ -111,7 +112,7 @@ describe('Navigation', async function() {
           assert.strictEqual(numNavigations, 6);
         }
 
-        assert.strictEqual(lhr.lighthouseVersion, '10.0.1');
+        assert.strictEqual(lhr.lighthouseVersion, packageJson.version);
```
Comment (brendankenny) [inline third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:115]: I assume the purpose of this is to assert that it's not accidentally testing the wrong version in DT, but let me know what you think of this solution, @adamraine :)

Comment (adamraine) [reply third-party/devtools-tests/e2e/lighthouse/navigation_test.ts:115]: This file get's rolled into DT frontend repo so this wouldn't work. We need to hardcode the version here.

Comment (brendankenny) [reply third-party/devtools-tests/e2e/lighthouse/navigation_test.ts]: OK, attempt #2 :)

**Comment by brendankenny** [comment]
Updated and should be good to go

**Review by paulirish — APPROVED** [review]

**Review by adamraine — APPROVED** [review]

**Comment by atifnimran** [comment]
O

**Pull request merged**